library/src/main/scala/scala/scalajs/js/special/package.scala

```diff
@@ -22,6 +22,14 @@ package scala.scalajs.js
  */
 package object special {
 
+  /** Dynamically tests whether a value is an instance of a JavaScript class.
+   *
+   *  This method is the exact equivalent of `x instanceof clazz` in
+   *  JavaScript.
```

Comment:
Could you provide a rationale why this is in `special`? All other methods have it an I think it is a very good thing.

Reply:
Done.